docs/templates/modules/hub.md

```diff
@@ -60,7 +60,8 @@ On this page:
 >   "type": "HubInstance",
 >   "version": "0.0.1",
 >   "learnMore": "https://aka.ms/finops/toolkit",
->   "exportScopes": <param:exportScopes>
+>   "exportScopes": <param:exportScopes>,
+>   "dataRetentionInMonths": 3
```

Comment:
@aromano2 I missed this in the last spec update. Can you either update your existing PR or add it to a new one when you close this one? Either way is fine. Please resolve this comment to ack.